oak_functions/examples/metrics/README.md

```diff
@@ -0,0 +1,46 @@
+# Differentially Private Metrics example
+
+This example show the use of differentially private metrics by a Wasm module.
+The client repeatedly sends either "a" or "b" to the server. The server reports
+event "a" when it receives "a", and does not report any events if it receives
+"b". A request still counts towards that batch count even if no events are
+reported.
+
+The resulting exported metrics should provide a relatively accurate count of how
+many times 'a' was received, but importantly it should not be possible to know
+which of the individual client sessions sent "a" by looking at the output. Even
+if an attacker knows what was sent in all the requests in the batch except for
+one, they should not be able to deduce with high certainty what was sent in the
+remaining request. The addition of Laplacian noise to the batch counts ensures
+this property, assuming appropriate settings are chosen.
+
+The allowed list of event labels, the privacy budget and the batch size are set
+in the server configuration file ('config.toml'). If multiple different event
+labels are allowed the budget would be evenly split across all of them.
+
+Using a privacy budget value (epsilon) of 1.0 and only one allowed label means
+that approximately 39% of the time the batch count will be accurate.
```

Comment:
Is the formula something that could be mentioned here, so that a reader has some idea of where the numbers are coming from?